drivers/hud_apr/app/models/hud_apr/generators/shared/fy2024/question_sheet.rb

```diff
@@ -0,0 +1,23 @@
+# utility class to dry up answers
+module HudApr::Generators::Shared::Fy2024
+  class QuestionSheet
+    attr_reader :report, :question
+    def initialize(report:, question:)
+      @report = report
+      @question = question
+    end
+
+    def update_cell_value(cell:, value:)
+      answer = @report.answer(question: question, cell: cell)
+      answer.update(summary: value)
+      answer
+    end
+
+    def update_cell_members(cell:, members:)
+      answer = @report.answer(question: question, cell: cell)
+      answer.add_members(members)
+      answer.update(summary: members.size)
+      answer
+    end
+  end
+end
```

Comment:
This looks great!  Way cleaner.  Thanks! 
(let's add the boiler plate at the top.)

```
###
 # Copyright 2016 - 2023 Green River Data Analysis, LLC
 #
 # License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
 ###
```